src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -22,7 +22,15 @@ class ColumnsEditorTab extends React.Component {
 
     return (
       <div className="scrivito_detail_content">
-        <Alignment widget={this.props.widget} />
+        <Alignment
+          alignment={this.props.widget.get("alignment")}
+          setAlignment={(alignment) => {
+            if (Scrivito.canWrite()) {
```

Comment:
* [ ] FYI I just realized that `canWrite` can be used in a non-connected context. Otherwise this handler should've been implemented with `Scrivito.load`.

Reply:
Do we document somewhere which functions need a loading context and which just works?

Reply:
In https://www.scrivito.com/js-sdk/load we state:

> CMS data (like any Obj, Widget or Binary) needs to be loaded from the cloud service before it can be used. When rendering content inside a React component, all CMS data is loaded automatically using Scrivito.connect.

So I think it's safe to assume non-CMS data (like user properties) to go without connect/load, except if explicitly documented like here https://www.scrivito.com/currenteditor-deaac83014976a73.

Reply:
The example of https://www.scrivito.com/js-sdk/canWrite does include a loading context (implicitly via `Scrivito.registerComponent`). So at least to me it does not seem obvious, that `canWrite` does not need a loading context.